ci/test.sh

```diff
@@ -2,13 +2,13 @@
 
 set -ex
 
-BEHAT_TAGS=$(php ci/behat-tags.php)
-
 # Run the functional tests for the framework
+BEHAT_TAGS=$(BEHAT_FEATURES_FOLDER=vendor/wp-cli/wp-cli/features php ci/behat-tags.php)
 vendor/bin/behat --format progress $BEHAT_TAGS --strict
 
 # Run the functional tests for the bundled commands
-REPOS=$(cat composer.json | grep -m 1 -oE "wp-cli/([a-z\-]*)-command")
+REPOS=$(cat vendor/wp-cli/wp-cli/composer.json | grep -oE "wp-cli/([a-z\-]*)-command")
 for REPO in $REPOS; do
+	BEHAT_TAGS=$(BEHAT_FEATURES_FOLDER=vendor/$REPO/features php ci/behat-tags.php)
```

Comment:
I don't understand what you mean. The behat tags script does not parse the `behat.yml` file.